libtransmission/session.cc

```diff
@@ -444,6 +445,7 @@ void tr_sessionGetSettings(tr_session const* s, tr_variant* d)
     tr_variantDictAddBool(d, TR_KEY_rpc_enabled, tr_sessionIsRPCEnabled(s));
     tr_variantDictAddStr(d, TR_KEY_rpc_password, tr_sessionGetRPCPassword(s));
     tr_variantDictAddInt(d, TR_KEY_rpc_port, tr_sessionGetRPCPort(s));
+    tr_variantDictAddInt(d, TR_KEY_rpc_socket_mode, tr_sessionGetRPCSocketMode(s));
```

Comment:
this could call `tr_sessionGetRPCSocketMode()`

Reply:
I assume you mean `tr_rpcGetRPCSocketMode`?